### PR TITLE
Deserialize postprocess

### LINKED ISSUE #3459: User exit: After deserialization perform additional steps
After deserialization, I would like to perform some additional checks/operations on some type of objects.
My scenario: If some abapGit based project has created BC Sets for customizing content then those BC Sets must be unpacked after deserialization(no manual step of activation via SCPR20).

### PR DESCRIPTION
**Pull request by goreraks**
closes #3459 
Postprocessing only make sense for TLOGO objects, thus the for IS_STEP-DDIC_ONLY = true objects are not included for postprocessing.

### CODE REVIEW

File: src/zcl_abapgit_objects.clas.abap [hunk 1]
```diff
@@ -784,6 +793,16 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
 
     zcl_abapgit_objects_activation=>activate( is_step-is_ddic ).
 
+    IF lt_postprocess_items[] IS NOT INITIAL.
+
+*     Call postprocessing
+      li_exit = zcl_abapgit_exit=>get_instance( ).
+
+      li_exit->deserialize_postprocess( it_items = lt_postprocess_items[]
```
Comment (larshp) [inline src/zcl_abapgit_objects.clas.abap:801]: just pass `is_step` to the exit? so there is no assumptions regarding abapGit implementation and future use

Comment (goreraks) [reply src/zcl_abapgit_objects.clas.abap:801]: Reason why I have introduced lt_postprocess_items is to capture the objects which were deserialized successfully.
May be I just remove this CHECK is_step-is_ddic = abap_false.??

Comment (larshp) [reply src/zcl_abapgit_objects.clas.abap:801]: successes and errors are recorded to the log, so the exit already knows the objects which were deserialized successfully

```
          ii_log->add_success( iv_msg = |Object { <ls_obj>-item-obj_name } imported|
                               is_item = <ls_obj>-item ).
```

Comment (goreraks) [reply src/zcl_abapgit_objects.clas.abap]: IS_STEP is passed for the postprocessing userexit.

**Comment by larshp** [comment]
There is a syntax error in the compiled report,
`ci/zabapgit_full.prog.abap[2854, 7] - Unknown type: Could not resolve top ZCL_ABAPGIT_OBJECTS=>TY_STEP_DATA`

the fix is to move the TY_STEP_DATA definition to ZIF_ABAPGIT_DEFINITIONS

**Comment by larshp** [comment]
I'm messing around, so I will fix the `Method definition "get_longtext" not found` error

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**